java/core/src/main/java/com/lancedb/lance/schema/SqlExpressions.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.lancedb.lance.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SqlExpressions {
```

Comment:
Can we add docstring for this class?
I.e., why we have `name` and `expression` in each object. 

Should we consider substrait or just use string for SQL?  @westonpace

Reply:
Add docs.